src/Grpc/HttpApi/src/Microsoft.AspNetCore.Grpc.HttpApi/Internal/Binding/HttpApiProviderSeviceBinder.cs

```diff
@@ -0,0 +1,282 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics.CodeAnalysis;
+using Google.Api;
+using Google.Protobuf.Reflection;
+using Grpc.AspNetCore.Server;
+using Grpc.AspNetCore.Server.Model;
+using Grpc.Core;
+using Grpc.Shared;
+using Grpc.Shared.Server;
+using Microsoft.AspNetCore.Grpc.HttpApi.Internal.CallHandlers;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Routing.Patterns;
+using Microsoft.Extensions.Logging;
+using MethodOptions = global::Grpc.Shared.Server.MethodOptions;
+
+namespace Microsoft.AspNetCore.Grpc.HttpApi.Internal.Binding;
+
+internal sealed partial class HttpApiProviderServiceBinder<TService> : ServiceBinderBase where TService : class
+{
+    private delegate (RequestDelegate RequestDelegate, List<object> Metadata) CreateRequestDelegate<TRequest, TResponse>(
+        Method<TRequest, TResponse> method,
+        string httpVerb,
```

Comment:
```suggestion
        HttpMethod httpMetho,
```

Fair to consider this as API feedback.

Reply:
HttpMethod is more of a HttpClient thing. In ASP.NET Core method is typically passed around as a string. In this case it needs to be a string for HttpMethodMetadata.